--- php-pecl-vld-0.8.0/vld-0.8.0/srm_oparray.c~	2005-01-19 12:59:54.000000000 +0100
+++ php-pecl-vld-0.8.0/vld-0.8.0/srm_oparray.c	2005-12-23 00:58:58.207336272 +0100
@@ -351,6 +351,7 @@
 			op->op2.op_type = VLD_IS_OPLINE;
 			break;
 
+#ifdef ZEND_JMP_NO_CTOR
 		case ZEND_JMP_NO_CTOR:
 			flags = OP2_USED;
 			if (op->op1.op_type != IS_UNUSED) {
@@ -361,6 +362,7 @@
 #endif
 			op->op2.op_type = VLD_IS_OPLINE;
 			break;
+#endif
 
 #ifdef ZEND_ENGINE_2
 		case ZEND_FETCH_CLASS:
